Source/Core/DolphinQt2/WiiUpdate.cpp

```diff
@@ -96,7 +96,10 @@ static WiiUtils::UpdateResult ShowProgress(QWidget* parent, Callable function, A
   auto* cancel_button = new QPushButton(QObject::tr("&Cancel"), parent);
   dialog.setCancelButton(cancel_button);
   Common::Flag was_cancelled;
-  QObject::disconnect(&dialog, &QProgressDialog::canceled, &dialog, &QProgressDialog::cancel);
+  // Disconnect the default cancel signal. We don't know if Qt connected it using the old
+  // or the new signal slot syntax, so we need to use the old syntax with a wildcard,
+  // which allows us to get rid of all slots in all cases.
+  QObject::disconnect(&dialog, SIGNAL(canceled()), nullptr, nullptr);
```

Comment:
Sorry, I totally missed that!

Still though: can `QObject::disconnect(&dialog, &QProgressDialog::canceled, 0, 0)` work here? It returns true on my computer (where the code in master right now returns false).